docs/api.swagger.yaml

```diff
@@ -1376,8 +1376,8 @@ paths:
   /warden/groups:
     get:
       description: >-
-        The subject making the request needs to be assigned to a policy
-        containing:
+        The subject making the request, if member is specified, needs to be
+        assigned to a policy containing:
```

Comment:
As discussed in my comment, replace this with one check to `rn:hydra:warden:groups`